bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,229 @@
+# airq Binding
+
+The airq Binding integrates the air analyzer <a href="http://www.air-q.com">air-Q</a> device into the openHAB system.
+
+With the binding, it is possible to subscribe to all data delivered by the air-Q device.
+
+<img src="https://uploads-ssl.webflow.com/5bd9feee2fb42232fe1d0196/5e4a8dc0e322ca33891b51e4_air-Q%20frontal-p-800.png" alt="air-Q image" width="400px" height="324px" />
+
+## Supported Things
+
+One only Thing is supported: the air-Q device.
```

Comment:
Why the `air-Q`?

Reply:
You used air-Q as the label for the thing definition, so this is what will show up in the main UI when adding the device. Not sure I understand what you are asking here.

Reply:
I mean the inverted comma, quote or however you call it, what surrounds air-Q in your suggestion. I can't write it in the text field because it starts a code section. You mean to use the quote around `air-Q` only when the device is meant, or whenever I use that name in the README.md?

Reply:
I was using it like italics, underlining the word. But that is not necessary, and if you dont like the use of it, just leave the quote out.